app/javascript/tagging/stories/tagging.stories.js

```diff
@@ -0,0 +1,266 @@
+import React from 'react';
```

Comment:
I don't think we have any use for `.stories.js` files here.

Reply:
Oh, good point.

But the docs do seem to be useful.. let's deal with storybook in a separate PR?
Moving more components soon, so I can finish that by setting up a ui "storybook", including for ui-components components.

Reply:
OK

Reply:
I've added a docs section to https://github.com/ManageIQ/manageiq-ui-classic/issues/6716 so as not to forget about this.